packages/kotti-ui/source/kotti-style/_layout.scss

```diff
@@ -22,7 +24,8 @@
 	margin-left: auto;
 	clear: both;
 
-	$grid-spacing: ($layout-spacing / ($layout-spacing * 0 + 1)) * $html-font-size;
+	$grid-spacing: math.div($layout-spacing, $layout-spacing * 0 + 1) *
```

Comment:
Unrelated, but does this divide even do anything?

I read this basically as:

1. ls / (ls * 0 + 1)
2. = ls / 1
3. = ls

Maybe I'm missing something, but this looks really fishy

Reply:
Good find.

However, I also don't find any usage of the classes that this variable defines (`grid-{xs,sm,md,lg,xl}`) So this is essentially dead anyway